### DISCUSSION TITLE
Special characters not displayed correctly in the WebGUI

### DISCUSSION THREAD

**Discussion by olegbliaher** [postalserver/postal#2576] [Help with using Postal]
Hi there, I noticed that on Postal 2.1.4, when I enter the following characters in the message body:

åäöÅÄÖéë

They are later displayed simply as ���������������� in the WebGUI.

In the raw database table, they are simply stored as `=C3=A5=C3=A4=C3=B6=C3=85=C3=84=C3=96=C3=A9=C3=AB=`

Is there any easy way to fix this? It's not a browser issue.

**Comment by pendurov** [comment]
I have same issue on Postal 2.1.4. Any ideas?

**Comment by Joe-Ng-1998** [comment]
Same here.

Non-English characters are not correctly displayed in Web GUI.

East Asian characters show as black blocks.
